### PR TITLE
On load check for annotation size cookie or config default

### PR DESCRIPTION
**Pull request by csolbs24**
# On load check for annotation size cookie or config default

## Description

Added that ability to change the default annotation size in the config. Additionally saves a cookie with the size of the annotations per subtask. On load, uses the cookie size for the annotations if it exists. If cookie doesn't exist, then uses the config's default. If neither exist, then if uses the size saved to the annotation like before this pull request.

## PR Checklist

- [ ] Merged latest main
- [ ] Version number in `package.json` has been bumped since last release
- [ ] Version numbers match between package `package.json` and `src/version.js`
- [ ] Ran `npm install` and `npm run build` AFTER bumping the version number
- [ ] Updated documentation if necessary (currently just in `api_spec.md`)
- [ ] Added changes to `changelog.md`

## Breaking API Changes

none

### CODE REVIEW

File: src/toolbox.ts [hunk 1]
```diff
@@ -441,9 +463,43 @@ export class AnnotationResizeItem extends ToolboxItem {
...
+            this.set_size_cookie(subtask.annotations.access[subtask.annotations.ordering[0]].line_size, subtask)
             return;
         }
-        return;
+        throw Error("Invalid Operation given to loop_through_annotations")
+    }
+
+    private set_size_cookie(cookie_value, subtask) {
+        let d = new Date();
+        d.setTime(d.getTime() + (10000 * 24 * 60 * 60 * 1000));
```
Comment (joshua-dean) [inline src/toolbox.ts:474]: what is the utility of this?

**Review by joshua-dean — APPROVED** [review]
LGTM 👍

File: src/toolbox.ts (hunk 1 above)
Comment (csolbs24) [reply src/toolbox.ts:474]: Sets when the cookie expires. In this example 100 years from when its created

Comment (joshua-dean) [reply src/toolbox.ts:474]: I will be filing a bug report in 2122

**Pull request merged**